.github/workflows/release-on-tag.yml

```diff
@@ -0,0 +1,35 @@
+name: Build & Publish FDroid APK
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  Gradle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: setup jdk
+        uses: actions/setup-java@v1
+        with:
+          java-version: 17
+      - name: Make Gradle executable
+        run: chmod +x ./gradlew
+      - name: Build Release APK
+        run: ./gradlew assembleNoinappRelease
+      - name: Sign APK
+        id: sign_apk
+        uses: r0adkll/sign-android-release@v1
+        with:
+          releaseDirectory: app/build/outputs/apk/noinapp/release
+          signingKeyBase64: ${{ secrets.SIGNING_WEB_KEYSTORE }}
+          alias: ${{ secrets.SIGNING_WEB_KEY_ALIAS }}
+          keyStorePassword: ${{ secrets.SIGNING_WEB_STORE_PASSWORD }}
+          keyPassword: ${{ secrets.SIGNING_WEB_KEY_PASSWORD }}
+      - name: Releasing using Hub
+        uses: kyze8439690/action-release-releaseapk@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOKEN_GITHUB }}
```

Comment:
what sort of permissions do you need for this? that token only has `read:packages`

Reply:
Found it! It requires `repo` access,  I'll need to create a new one.

Reply:
Added new secret and replaced name in the yaml